src/main/java/app/linguistai/bmvp/service/wordbank/UnknownWordService.java

```diff
@@ -343,6 +344,45 @@ public ROwnerUnknownWordList deleteList(UUID listId, String email) throws Except
         }
     }
 
+    @Override
+    @Transactional
+    public UnknownWordList getRandomActiveUnknownWordList(UUID userId) throws Exception {
+        try {
+            List<UnknownWordList> activeLists = listRepository.findByUserId(userId)
+                .stream()
+                .filter(UnknownWordList::getIsActive)
+                .toList();
+
+            if (activeLists.isEmpty()) {
+                throw new NotFoundException("No active unknown word lists found for user with ID: [" + userId + "].");
+            }
```

Comment:
This exception results in a 404 error code. So when user logs in, assignQuests are called and if there are no active lists then I believe log-in endpoint returns 404. Can you try if this is the case?

Reply:
Tested with a new user and confirmed no active word list exists, login is successful. However 400 (bad request) returns when a new UseWord quest is attempted to be given to the user.

Since the user must have at lease one active word list at one time (rule), this should give an error in my opinion.